project.rosinstall

```diff
@@ -17,3 +17,8 @@
     local-name: lma_kinematics_plugin
     uri: 'https://github.com/shadow-robot/lma_kinematics_plugin'
     version: F_ik_with_joint_limits
+
```

Comment:
will also need the ros_control branch for this to compile
https://github.com/CentroEPiaggio/ros_control/tree/multi-robot-test